public/app/ui/pages/about/AboutPage.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'ui/responsive'
+        ],
+        function (responsive) {
+            return function () {
+                this.ready = function (element, callback) {
+                    responsive.update();
```

Comment:
So calling `responsive.update()` here means, "after the page has loaded and been rendered, make sure the footer is in the right place"